Makefile

```diff
@@ -82,18 +82,22 @@ unit-tests:
 e2e-tests: prepare-e2e-tests e2e-tests-smoke e2e-tests-cassandra e2e-tests-es e2e-tests-self-provisioned-es e2e-tests-streaming e2e-tests-examples1 e2e-tests-examples2
 
 .PHONY: prepare-e2e-tests
-prepare-e2e-tests: crd build docker push
+prepare-e2e-tests: e2e-tests-prepare-global-manifests crd build docker push
 	@mkdir -p deploy/test
-	@cp test/role_binding.yaml deploy/test/namespace-manifests.yaml
+	@cp test/service_account.yaml deploy/test/namespace-manifests.yaml
 	@echo "---" >> deploy/test/namespace-manifests.yaml
 
-	@cat test/role.yaml >> deploy/test/namespace-manifests.yaml
-	@echo "---" >> deploy/test/namespace-manifests.yaml
+	@cat test/operator.yaml | sed "s~image: jaegertracing\/jaeger-operator\:.*~image: $(BUILD_IMAGE)~gi" >> deploy/test/namespace-manifests.yaml
 
-	@cat test/service_account.yaml >> deploy/test/namespace-manifests.yaml
-	@echo "---" >> deploy/test/namespace-manifests.yaml
+.PHONY: e2e-tetsts-prepare-global-manifests
+e2e-tests-prepare-global-manifests: crd build docker push
```

Comment:
If I run `make e2e-tests-prepare-global-manifest`, should I expect `crd build docker push` to also be executed?

Reply:
No, I will actually merge the global-manifests target with the prepare e2e test. It's not being reused in other places so a single target is enough.